### DISCUSSION TITLE
Serve initial index.html

### DISCUSSION THREAD

**Discussion by SavageStyle** [Theldus/wsServer#58] [Q&A] [answered]
I can not find the built in way to serve initial index file to the client. Could you please post an example? That would be great include that in wsServer.

**Comment by Theldus** [comment, marked as answer]
Hi @SavageStyle,
Thanks for giving wsServer a try,

Unfortunately wsServer only provides a WebSocket server, not HTTP.

I don't know what your needs are, but you can always make your project's HTML available for the user to open locally. If not suitable, you can use a web server like Apache or Nginx for this.

(There are even servers with a single .c, like [Althttpd](https://sqlite.org/althttpd/doc/trunk/althttpd.md) (used on the SQLite site), which I highly recommend looking at).

If you want everything in one place, I'm sure there is some small library for HTTP servers in C.
